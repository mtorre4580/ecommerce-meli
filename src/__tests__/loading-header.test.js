import React from 'react';
import LoadingHeader from '../components/detail/loading-header/loading-header';
import renderer from 'react-test-renderer';

describe('<LoadingHeader />', () => {

    it('should render snapshot', () => {
        const component = renderer.create(<LoadingHeader />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

 });



