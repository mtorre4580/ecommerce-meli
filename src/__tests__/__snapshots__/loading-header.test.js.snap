// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingHeader /> should render snapshot 1`] = `
<div
  className="ctn-loader container"
>
  <div
    className="loader-detail placeholder-loader"
  >
    <div
      className="content-loader"
    >
      <div
        className="full-line animated-background"
      />
      <div
        className="full-line animated-background"
      />
      <div
        className="half-line animated-background"
      />
    </div>
  </div>
</div>
`;
